src/components/calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div id="calculator">
+        <input type="text" value={0} className="screen" />
+        <div id="numbers">
+          <button type="button" id="AC" value="AC" onClick={null}>Ac</button>
+          <button type="button" id="+/-" value="+/-" onClick={null}>+/-</button>
+          <button type="button" id="%" value="%" onClick={null}>%</button>
+          <button type="button" id="+" value="+" className="special" onClick={null}>&divide;</button>
```

Comment:
[Optional] This is one of the project requirements but was missed by the previous reviewer so I highly encourage you to implement it.

- According to the project requirements, your calculator component should have at least 1 child component. Please try to create and import an additional component for use in your calculator component.
<img width="293" alt="image" src="https://user-images.githubusercontent.com/37116322/236634008-ed7c41bf-3e80-42fc-92dc-200097b9117b.png">